tests/test_ram/test_ram.py

```diff
@@ -0,0 +1,362 @@
+import time
+from datetime import datetime
+
+import boto3
+import sure  # noqa
+from botocore.exceptions import ClientError
+from nose.tools import assert_raises
+
+from moto import mock_ram, mock_organizations
+
+
+@mock_ram
+def test_create_resource_share():
+    # given
+    client = boto3.client("ram", region_name="us-east-1")
+
+    # when
+    response = client.create_resource_share(name="test")
+
+    # then
+    resource = response["resourceShare"]
+    resource["allowExternalPrincipals"].should.be.ok
+    resource["creationTime"].should.be.a(datetime)
+    resource["lastUpdatedTime"].should.be.a(datetime)
+    resource["name"].should.equal("test")
+    resource["owningAccountId"].should.equal("123456789012")
```

Comment:
You can import the ACCOUNT_ID from `moto.core`, no need to hardcode it everytime

Reply:
Good point, I got used to hardcode the results of my tests.